app/views/shared/_plan_shopping_progress.html.erb

```diff
@@ -0,0 +1,16 @@
+<% step = (defined? step) ? step.to_i : 1 %>
+<% ridp = (defined? ridp) ? ridp : false %>
```

Comment:
are either `continue_disabled` or `ridp` being used here? If not, I think we can remove them...

and if we do remove them, I'm thinking this partial ends up looking exactly like the `consumer_signup` partial we have, save for the "title" label. would it be worth combining these two partials into one "progress_nav" partial or something to that effect?